ark/jsonschema/README.md

```diff
@@ -0,0 +1,51 @@
+# @ark/jsonschema
```

Comment:
Thanks for writing this up! At some point we'll want to add an `arktype.io` docs section, but that's a problem for another day

Reply:
Fwiw I'm happy to help out with doc-writing, but outside of this PR, and I'd need some guidance as to format etc..